public/app/plugins/datasource/jaeger/components/ConfigEditor.tsx

```diff
@@ -18,6 +18,14 @@ export const ConfigEditor = ({ options, onOptionsChange }: Props) => {
 
   return (
     <div className={styles.container}>
+      <DataSourceDescription
+        dataSourceName="Jaeger"
+        docsLink="https://grafana.com/docs/grafana/latest/datasources/jaeger"
+        hasRequiredFields={false}
```

Comment:
Will probably update this when adding new auth component to Tempo/Jaeger/Zipkin configs.